server/job-scheduler.js

```diff
@@ -32,14 +33,59 @@ class JobScheduler {
 
   async bootstrap() {
     await this.scheduleActiveCronJobs();
-    
+  
     await this.scheduleJobStatusPolling();
   }
 
+  //Handle Manual Job 
+  // This function sends email notification
+  // and record job execution
+  async handleManualJobScheduling (job){
+    let notificationOptions = {
+      for : 'manaulJob',
+      jobName : job.name,
+      jobTitle: job.title,
+      contact: job.contact,
+      url : `${process.env.WEB_URI}${job.application_id}/manualJobDetails/${job.id}`
+    }
+
+    const result = await notificationMoudle.sendEmailNotification(notificationOptions)
```

Comment:
Do we really need a new method for sending email notification. You can for the Subject and Content of the email and send it to the generic email notification function instead of creating a new one?